docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,147 @@
+==============
+Atop connector
+==============
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System and Process Monitor) Linux server performance analysis tool.
+
+Requirements
+------------
+
+In order to use this connector, the machine on which the Trino worker is running
+needs to have the ``atop`` tool installed locally.
+
+The system utilization statistics are internally read from
+``/var/log/atop/atop_YYYYMMDD`` files which correspond to the each day in the last
+``atop.max-history-days`` days. These files contain compressed binary data written
+on a ``10`` minutes interval by ``/etc/atop/atop.daily`` cron daemon.
```

Comment:
If this is just the default then do mention so (or maybe omit altogether) since distributions and versions may have different defaults. We aren't here to document atop - just the connector.

Reply:
Indeed. This detail is not really relevant for Trino.